webapp-django/crashstats/cron/admin.py

```diff
@@ -41,6 +41,7 @@ class LogAdmin(admin.ModelAdmin):
         'success',
         'pretty_error'
     ]
+    list_filter = ['app_name']
```

Comment:
This makes it easier to look at all the log entries for a specific app. Makes it easier to diagnose issues.